packages/appium/lib/constants.js

```diff
@@ -33,22 +39,40 @@ export const KNOWN_PLUGINS = Object.freeze(
   }),
 );
 
-// This is a map of driver names to npm packages representing those drivers.
-// The drivers in this list will be available to the CLI so users can just
-// type 'appium driver install 'name'', rather than having to specify the full
-// npm package. I.e., these are the officially recognized drivers.
-export const KNOWN_DRIVERS = Object.freeze(
+export const MOBILE_DRIVERS = Object.freeze(
   /** @type {const} */ ({
     uiautomator2: 'appium-uiautomator2-driver',
     xcuitest: 'appium-xcuitest-driver',
-    mac2: 'appium-mac2-driver',
     espresso: 'appium-espresso-driver',
+  }),
+);
+
+export const DESKTOP_DRIVERS = Object.freeze(
```

Comment:
do we also want to include the windows driver?

Reply:
Maybe we removed Win as w3c spec compatibility matter before. I'm ok to bring it back though.

Reply:
maybe we could just install it via --source npm

@jlipps What do you think?

Reply:
When we bring windows driver here, https://appium.io/docs/en/latest/ecosystem/drivers/ also needs to be updated. The modification could be done later with the documentation update PR.

Reply:
I think it's fine if the Windows driver is added as a 'known' driver, but it would be cleaner to have it changed in a separate PR. Then it might be worth to also add something like a warning about the lack of w3c compatibility when installing it.

Reply:
same question probably applies to the flutter driver in mobile context

Reply:
Personally, flutter driver should keep not in this list since it depends on flutter_driver API, which is still under discussion in the flutter team if it should keep be maintained, invest more or end the life. I've seen various discussions in Flutetr repo about this, thus appium flutter driver is a good option but maybe can be an option.

I'd recommend uia2 and xcuitest with black box approach if possible. Flutter 3.19 can support resource-id/accessibility identifier method as well, thus they are more uia2/xcuitest driver friendly.

Reply:
-> https://github.com/appium/appium/issues/20124

Reply:
seems like maybe we don't include the windows driver given it's not maintained?